public/viewtopic.php

```diff
@@ -207,7 +207,7 @@
             echo userAvatar($nextCommentAuthor, label: false, iconSize: 64);
             echo "</td>";
 
-            echo "<td class='w-full py-3' id='$nextCommentID'>";
+            echo "<td class='w-full py-3 break-normal' id='$nextCommentID'>";
```

Comment:
This doesn't seem to have any effect in Chrome or Edge. It does seem to work in Firefox.

`break-all` does appear to work in all three, but I don't now the implications of using it, or if there's a better middle ground.

Reply:
`break-all` should be fine, although I am seeing `break-normal` work on Chrome and Edge. Is it possible on your local machine that Tailwind isn't picking up & adding the class name to the compiled CSS?

Reply:
Anything is possible, but running `npm run build` doesn't have any noticeable effect when using `break-normal`, but was required when using `break-all`.

Reply:
And to echo my previous statement, it did work in Firefox, so that seems to have picked it up.

Reply:
Confirmed this was an issue with Tailwind & Vite on my local setup. Fixed.